base/sg_cluster_cfg.go

```diff
@@ -66,14 +68,18 @@ func (c *CfgSG) Get(cfgKey string, cas uint64) (
 func (c *CfgSG) Set(cfgKey string, val []byte, cas uint64) (uint64, error) {
 
 	InfofCtx(c.loggingCtx, KeyDCP, "cfg_sg: Set, key: %s, cas: %d", cfgKey, cas)
-	var casOut gocb.Cas
 	var err error
 	bucketKey := sgCfgBucketKey(cfgKey)
-	if cas == 0 {
-		casOut, err = c.bucket.Insert(bucketKey, val, 0)
-	} else {
-		casOut, err = c.bucket.Replace(bucketKey, val, gocb.Cas(cas), 0)
-	}
+
+	casOut, err := c.bucket.WriteCas(bucketKey, 0, 0, cas, val, 0)
+
+	/*
+		if cas == 0 {
+			casOut, err = c.bucket.Insert(bucketKey, val, 0)
+		} else {
+			casOut, err = c.bucket.Replace(bucketKey, val, gocb.Cas(cas), 0)
+		}
+	*/
```

Comment:
The WriteCas replacement looks good, don't think we need this any more.

```suggestion
```

Reply:
Oops - thanks